src/main/features/response/views/statement-of-means/bank-accounts.njk

```diff
@@ -0,0 +1,108 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, poundInput, linkButton, saveAndContinueButton, errorSummary %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('List your bank and savings accounts') %}
+
+{% macro bankAccountRow(index, form) %}
+
+  {% set prefix = 'rows[' + index + ']' %}
+  {% set typeOfAccountKey = prefix + '[typeOfAccount]' %}
+  {% set typeOfAccountValue = form.valueFor(typeOfAccountKey) %}
+  {% set typeOfAccountError = form.errorFor(typeOfAccountKey) %}
+
+  {% set isJointKey = prefix + '[isJoint]' %}
+  {% set isJointValue = form.valueFor(isJointKey) %}
+  {% set isJointError = form.errorFor(isJointKey) %}
+
+  {% set balanceKey = prefix + '[balance]' %}
+  {% set balanceValue = form.valueFor(balanceKey) %}
+  {% set balanceError = form.errorFor(balanceKey) %}
+
+  {% set anyErrorOccurred = balanceError or typeOfAccountError or isJointError %}
+  <div class="triple-column{% if anyErrorOccurred %} form-group-error {% endif %}">
+
+    {% if typeOfAccountError %}<span id="{{ typeOfAccountKey }}[label]" class="error-message">{{ t(typeOfAccountError) }}</span>{% endif %}
+    {% if isJointError %}<span id="{{ isJointKey }}[label]" class="error-message">{{ t(isJointError) }}</span>{% endif %}
+    {% if balanceError %}<span id="{{ balanceKey }}[label]" class="error-message">{{ t(balanceError) }}</span>{% endif %}
+
+    <label class="form-label-bold mobile-show" for="{{ typeOfAccountKey }}">{{ index + 1 }} {{ t('. Type of account') }}</label>
+    <select name="{{ typeOfAccountKey }}" id="{{ typeOfAccountKey }}" class="form-control column-1{% if typeOfAccountError %} form-group-error {% endif %}" style="width:30%">
+      <option value="">Choose</option>
```

Comment:
We should translate select options. Here and all options below.

Reply:
ok